indigo_app/templates/indigo_app/place/_bulk_approve_form.html

```diff
@@ -62,8 +62,15 @@ <h5 class="card-header">
                 <ul>
                   {% for work in form.broker.import_task_works %}
                     <li>
+                      {% if work in form.broker.missing_import_date_works %}
+                        <i class="fas fa-exclamation-triangle text-warning"></i>
+                      {% endif %}
                       <a target="_blank" href="{% url 'work' work.frbr_uri %}">{{ work.title }}</a>
                       <span class="text-muted">{{ work.frbr_uri }}</span>
+                      {% if work in form.broker.missing_import_date_works %}
+                        <i class="fas fa-exclamation-triangle text-warning"></i>
+                        <span class="alert alert-danger p-1">Import date missing</span>
```

Comment:
translateable. Perhaps include an indication of how the problem can be resolved: "Import date missing - add a publication date, commencement date or consolidation date."